### PR TITLE
App version

### LINKED ISSUE #26: Display git version on bot
It would be nice to have a way to see the version of the code that's running on the bot. 

Maybe an !about command or put it in the !help

### PR DESCRIPTION
**Pull request by adamschachne**

### CODE REVIEW

**Comment by adamschachne** [comment]
image

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -6,22 +6,30 @@ on:
...
+      - name: Set version
+        id: version
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
       # Build the image with tag matching repo owner name
-    - uses: actions/checkout@v3
-    - name: Build the Docker image
-      run: docker build -t ghcr.io/${{ github.repository }} .
+      - name: Build the Docker image
+        run: docker build --build-arg VERSION=${{ steps.version.outputs.sha_short }} -t ghcr.io/${{ github.repository }} .
```
Comment (classBabacar) [inline .github/workflows/docker-image.yml:31]: I was thinking of using a more sequential number like 1.0.0 then 1.0.1 as we go deeper, what does a45a2b4 do for us? Looking at the code how do you know we are using version a45a2b4?

**Review by classBabacar — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (classBabacar) [reply .github/workflows/docker-image.yml:31]: like {{github.run_id}} so we can can just look at the pipelines and see the latest version deployed

Comment (adamschachne) [reply .github/workflows/docker-image.yml:31]: We shouldn't overcomplicate it. A short hash tells us exactly which revision was deployed, like in those links.

Comment (classBabacar) [reply .github/workflows/docker-image.yml:31]: Isnt this overcomplicating it though: ```echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"```
when you can just do ```{{github.sha}}```

**Review by classBabacar — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (adamschachne) [reply .github/workflows/docker-image.yml:31]: Sure, that is simpler. It should still be shortened to 7 chars when the bot sends it, though. 40 is too long

Comment (classBabacar) [reply .github/workflows/docker-image.yml:31]: hmm, it makes practical sense, for me personally I just like simpler things, so later down the road we aren't looking at this command trying to figure out whats going on, but we can push if you want

**Review by classBabacar — APPROVED** [review]

**Pull request merged**